### PR TITLE
SPOI-10697 Kafka to Kafka with transform application template.

### PR DESCRIPTION
**Pull request by deepak-narkhede**
Kafka to Kafka with transform application template.

### CODE REVIEW

**Comment by deepak-narkhede** [comment]
Either one of you, please review. @yogidevendra or @chaithu14 or @shubham-pathak22

**Comment by deepak-narkhede** [comment]
@francisf . Please review.

File: kafka-to-kafka-transform/src/main/resources/META-INF/properties.xml [hunk 1]
```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0"?>
+<configuration>
```
Comment (francisf) [inline kafka-to-kafka-transform/src/main/resources/META-INF/properties.xml]: License needed?

File: kafka-to-kafka-transform/src/assemble/appPackage.xml [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<assembly xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (francisf) [inline kafka-to-kafka-transform/src/assemble/appPackage.xml]: License required?

File: kafka-to-kafka-transform/pom.xml [hunk 3]
```diff
@@ -0,0 +1,315 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (francisf) [inline kafka-to-kafka-transform/pom.xml]: license

File: kafka-to-kafka-transform/pom.xml [hunk 4]
```diff
@@ -0,0 +1,315 @@
...
+      </exclusions>
+      -->
+    </dependency>
+    <dependency>
+      <groupId>org.apache.kafka</groupId>
+      <artifactId>kafka_2.11</artifactId>
+      <version>0.9.0.1</version>
+    </dependency>
+    <dependency>
+      <groupId>net.sf.supercsv</groupId>
```
Comment (francisf) [inline kafka-to-kafka-transform/pom.xml]: csv dependency required?

File: kafka-to-kafka-transform/README.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
+### Description
+The Kafka process kafka application ingest data from kafka, process them and write back to kafka. This application uses PoJoEvent as an example schema, this can be customized to use custom schema based on specific needs.
```
Comment (francisf) [inline kafka-to-kafka-transform/README.md]: ingest(s)

File: kafka-to-kafka-transform/src/main/resources/META-INF/properties.xml (hunk 1 above)
Comment (deepak-narkhede) [inline kafka-to-kafka-transform/src/main/resources/META-INF/properties.xml]: Done.

File: kafka-to-kafka-transform/src/assemble/appPackage.xml (hunk 2 above)
Comment (deepak-narkhede) [inline kafka-to-kafka-transform/src/assemble/appPackage.xml]: not required.

File: kafka-to-kafka-transform/pom.xml (hunk 3 above)
Comment (deepak-narkhede) [inline kafka-to-kafka-transform/pom.xml]: not required.

File: kafka-to-kafka-transform/pom.xml (hunk 4 above)
Comment (deepak-narkhede) [inline kafka-to-kafka-transform/pom.xml]: Yes.

File: kafka-to-kafka-transform/README.md (hunk 5 above)
Comment (deepak-narkhede) [inline kafka-to-kafka-transform/README.md]: Done.

**Comment by deepak-narkhede** [comment]
@francisf . Incorporated the comments. Please have a look.

**Comment by francisf** [comment]
lgtm

**Pull request merged**